lib/features/issue/presentation/view/issue_room_screen.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:debateseason_frontend_v1/core/constants/dimensions.dart';
+import 'package:debateseason_frontend_v1/core/constants/gaps.dart';
+import 'package:debateseason_frontend_v1/core/constants/text_style.dart';
+import 'package:debateseason_frontend_v1/features/issue/presentation/widgets/issue_card.dart';
+import 'package:debateseason_frontend_v1/widgets/de_app_bar.dart';
+import 'package:debateseason_frontend_v1/widgets/de_gesture_detector.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:get/get.dart';
+
+import 'package:debateseason_frontend_v1/widgets/de_scaffold.dart';
+import 'package:debateseason_frontend_v1/core/constants/color.dart';
+import 'package:debateseason_frontend_v1/widgets/de_text.dart';
+import 'package:debateseason_frontend_v1/features/issue/presentation//view_model/issue_room_view_model.dart';
+
+class IssueRoomScreen extends GetView<IssueRoomViewModel> {
+  const IssueRoomScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return DeScaffold(
+      appBar: _appBar(),
+      body: _body(),
+    );
+  }
+
+  DeAppBar _appBar() {
+    return DeAppBar(
+      title: '이슈',
+      isBack: true,
+      isCenter: false,
+    );
+  }
+
+  Widget _body() {
+    return Padding(
+      padding: EdgeInsets.fromLTRB(20, 12, 20, 0),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          _newChatCount(),
+          Gaps.v40,
+          _joinedCommunities(),
+          Gaps.v40,
+          Expanded(
+            child: _debateView(),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _newChatCount() {
+    return Container(
+      padding: Dimensions.vertical12,
+      decoration: ShapeDecoration(
+        color: grey120,
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(12.0),
+        ),
+      ),
+      child: Column(
+        children: [
+          DeText(
+            '오늘 신규 대화',
+            style: body16M.copyWith(color: grey50),
+          ),
+          Gaps.v12,
+          DeText(
+            '300개',
+            style: body16Sb.copyWith(color: grey10),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _joinedCommunities() {
+    Widget comm() {
+      return Container(
+        width: 36,
+        height: 36,
+        margin: EdgeInsets.fromLTRB(0, 0, 8, 0),
```

Comment:
```suggestion
        margin: EdgeInsets.only(right: 8),
```
이렇게도 가능합니다. 가독성은 뭐가 나은지 모르겠네요 only를 아예 안쓰고 fromLTRB로 하는게 가독성이 더 좋을 수 있을 것 같아요 ㅋㅎㅋㅋㅋ

Reply:
오 가독성으로보면 fromLTRB로 통일이 좋을 것 같아요.
근데 쓰기엔 저게 넘 쉽고 직관적일 것 같네여ㅋㅋㅋㅋ 좋은 거 배웠습니다